kleva-j/iReporter
# Feature/#162811186/request all records from frontend

#### What does this PR do?
- Ensures an authenticated user's records are requested from the frontend

#### Description of the task to be completed
 - create functions to make requests to the API from the frontend to get all records

#### How should this be manually tested?
- Clone the repository
- Check out into the branch Feature/#162811186/request-all-records-from-frontend
- Install all dependencies
- run `npm start` to run the app successfully.

#### What are the relevant pivotal tracker stories?
[162811186](https://www.pivotaltracker.com/story/show/162811186)
